composeApp/src/commonMain/kotlin/com/developersbreach/kotlindictionarymultiplatform/ui/screens/topic/TopicCard.kt

```diff
@@ -63,7 +61,12 @@ fun TopicCard(
                     .background(MaterialTheme.colorScheme.primary, CircleShape),
                 contentAlignment = Alignment.Center,
             ) {
-                Icon(Icons.Filled.Search, contentDescription = stringResource(Res.string.icon))
+                Text(
+                    text = topic.firstOrNull()?.uppercase() ?: "?",
```

Comment:
Composables should not be having any such computations, such transformations needs to be handled in outside the ViewModel.